docs/cost/index.md

```diff
@@ -33,30 +33,73 @@ You can read more about the Infrastructure Data Standard in [CoST Guidance Note
 
 ```
 
-## Mapping against the IDS
+## Mapping to IDS and from OCDS
 
 <!-- Note - mappings come from https://docs.google.com/spreadsheets/d/1tpXKCrNY1vUEPo6O1j-GPhxgSna7CZ5uwz_eTNLEOr8/edit#gid=2054628701 -->
 
-The following mapping tables describe how each element of the CoST Infrastructure Data Standard can be represented as **structured data** using the [Project-Level Data Specification](../projects/index.md) and contracting process information embedded within the project-level data.
+The following mapping tables describe:
 
-In cases where OCDS data is available, many of the contracting process fields within a project-level data file can also be **automatically populated** using the OCDS data.
+* How each element of the CoST Infrastructure Data Standard can be represented as **structured data** using the [Project-Level Data Specification](../projects/index.md), in the 'Mapping to OC for Infrastructure' column.
+
+* How existing OCDS data can be used to populate project-level and contracting process summary data, in the 'Mapping from OCDS' column.
+
+```eval_rst
+.. admonition:: Mapping from OCDS
+    :class: Note
+
+    .. markdown::
+
+        Some mappings use fields from [OCDS extensions](http://standard.open-contracting.org/latest/en/extensions/), in these cases the name of the extension is noted in brackets and, where possible, alternative mappings are provided which use only fields from the core OCDS schema.
```

Comment:
Run-on sentence. brackets -> parentheses ('brackets' typically refers to square brackets). Inconsistent pluralization. Fixed version:

> Some mappings use fields from [OCDS extensions](http://standard.open-contracting.org/latest/en/extensions/). In these cases, the names of extensions are noted in parentheses; where possible, alternative mappings are provided that use only fields from the core OCDS schema.